dradis/dradis-ce
# Fix fancy quotes

### Spec
We're converting the straight quotes to fancy quotes in the Web UI and in Word exports.

**Proposed solution**
Update the textile formatters to use non-fancy quotes instead.

### How to test

1. Update an issue and add single and double quotes in the content.
2. Click "Preview" and confirm that the quotes were not converted.
3. Export the project.
4. Confirm that the produced report does not have fancy quotes.

### Check List
